site/search-javascript.njk

```diff
@@ -0,0 +1,10 @@
+---
+permalink: search.js
+eleventyExcludeFromCollections: true
+---
+const searchItems = {% include "../static/searchData.json" %};
+
+{% set js %}
+  {% include "js/search.js" %}
+{% endset %}
+{{ js | safe }}
```

Comment:
Can do `jsmin` here to reduce the JS size down.

Reply:
Yeah, we should def do that. :)